src/sentry/web/frontend/setup_wizard.py

```diff
@@ -143,6 +134,75 @@ def get(self, request: HttpRequest, wizard_hash) -> HttpResponseBase:
         context["organizations"] = list(org_mappings_map.values())
         return render_to_response("sentry/setup-wizard.html", context, request)
 
+    def post(self, request: HttpRequest, wizard_hash=None) -> HttpResponse:
+        """
+        This updates the cache content for a specific hash
+        """
+        organization_id = request.POST.get("organizationId", None)
+        project_id = request.POST.get("projectId", None)
+
+        if organization_id is None or project_id is None or wizard_hash is None:
+            return HttpResponseBadRequest()
+
+        mapping = get_object_or_404(OrganizationMapping, organization_id=organization_id)
+
+        project = project_service.get_by_id(organization_id=mapping.organization_id, id=project_id)
```

Comment:
We should also check if user has access to the project, I am currently trying to figure out what is the best way to do this here

Reply:
Even without team / project membership users are able to see which projects exist + the dsn is not a secret.
The current wizard already displays all of them and I would not like to change this behaviour in this PR.
I will create a follow-up ticket to re-discuss it.

Reply:
There are organizations that do not make all projects linked to an organization transparent to all users in the org ([see here for example](https://github.com/getsentry/sentry/blob/13916656bbb9e6bfebe1f52a9ddde9f25a81dfc4/src/sentry/api/bases/organization.py#L399-L434)) - behaviour like this can potentially leak project existence to users that should not have access to it

Reply:
but as far as I can see the `<organization>/projects` endpoint does not implement this, or am I missing something?
https://github.com/getsentry/sentry/blob/ef63fb3c610504b39d484257195e034b558a0a7d/src/sentry/api/endpoints/organization_projects.py#L66

Reply:
Let me check this - I'm pretty sure that is the intended behaviour

Reply:
Just to leave a record here after an offline discussion: it seems the endpoint does indeed return all projects, therefore this PR does not need to change this behaviour - I will check whether this is intended and potentially make sure we clean it up if needed.